nntrainer/tensor/manager.cpp

```diff
@@ -355,9 +355,8 @@ std::vector<Weight *> Manager::requestWeights(
     node.getExecutionOrder();
 
   std::vector<unsigned int> var_exec_order(
-    {forwarding_order, calcGradient_order, calcDerivative_order});
-  std::vector<unsigned int> default_grad_exec_order(
-    {calcGradient_order, calcDerivative_order});
+    {forwarding_order, calcDerivative_order});
```

Comment:
Can you please describe why did you remove calcGradient_order from var_exec_order?
I think weights are also needed in some cases when calculate gradient.
Please check https://github.com/nnstreamer/nntrainer/blob/main/nntrainer/layers/grucell.cpp#L472